### ISSUE TITLE
Typo in klipper toleration pod (node-role.kubernetes.io/master=effect:NoSchedule) 

### ISSUE THREAD

**Issue by louiznk** [rancher/k3s#1989]
**Environmental Info:**
K3s Version: v1.17.4

Node(s) CPU architecture, OS, and Version: All

Cluster Configuration: 1 master 3 node but no link to this

**Describe the bug:**
There is a typo on the klipper daemonset toleration spec. It may be my typo.

The toleration of the pod(s) are
```
...spec.tolerations:
   - key: noderole.kubernetes.io/master
....
``` 
the key must be node-role.kubernetes.io/master (instead of noderole.kubernetes.io/master)

If you taint your master node with 
```
node-role.kubernetes.io/master=effect:NoSchedule
```
The svclb-traefik-ingress-controller daemonset will not be deploy on the master node.

**Steps To Reproduce:**

- Installed K3s: 
- taint the <master-node>
```
kubectl taint nodes <master-node>  node-role.kubernetes.io/master=effect:NoSchedule
```
- restart k3s, the traefik ingress will not be accessible on the master

**Expected behavior:**

Can taint the master node with the classical  node-role.kubernetes.io/master=effect:NoSchedule

**Actual behavior:**

When you taint the master node traefik won't be accessible because klipper is not deploy.

**Additional context / logs:**

I will submit a PR for this typo fix.

**Comment by louiznk** [comment]
It's could be the the same issue than #1988

**Comment by dweomer** [comment]
Addressed by #1892
